src/features/court/index.jsx

```diff
@@ -0,0 +1,51 @@
+import React, { useEffect } from 'react';
+
+import { HalfCourt } from './components/half-court';
+import { SubCourt } from './components/sub-court';
+
+import styles from './index.module.scss';
+
+/** @type {React.FC} */
+export const Court = () => {
+  useEffect(() => {
+    const onScreenChange = () => {
+      const bodyStyle = document.body.style;
+      const width = Math.floor(0.8 * document.body.clientWidth / 2) * 2;
+      const height = Math.floor(315 / 680 * width / 2) * 2;
+      bodyStyle.setProperty('--court_width', `${width}px`);
+      bodyStyle.setProperty('--court_height', `${height}px`);
+    }
+
+    onScreenChange();
+    window.addEventListener('resize', onScreenChange);
+
+    return () => {
+      window.removeEventListener('resize', onScreenChange);
+    };
+  });
```

Comment:
I used `useEffect` and event listener for some styling effect as well, but the animation / response is not much smooth than pure CSS. Do you experience similarly?

Reply:
@kavimaluskam no idea🤔
It looks fine for me.
What do you mean by pure CSS vs `useEffect` one?
Is there an example?